sass/style.sass

```diff
@@ -1227,3 +1227,50 @@ textarea.deploy-key
 .content .variables-description p
 	max-width: none
 
+button.abort
```

Comment:
Nah, I want it margin-less so it fits inline with the label on the left, and really small, so it doesn't collapse the log. I don't want to fix up the entire page :-| WILL_NOT_PERFORM?